contracts/community-nft1.0.sol

```diff
@@ -0,0 +1,297 @@
+pragma solidity ^0.6.2;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "../utils/StringUtils.sol";
+import "../SponsorWhitelistControl.sol";
+import "../owner/Operator.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+
+contract TspaceAllnighter is ERC1155, Operator {
```

Comment:
This repo is common for all projects as OZ did, so please avoid any concrete project name.

On the other hand, I think the best place for such contract is `contracts/token/CRC1155/presets`, you could refer to the code structure of OZ.

You could refer to the OZ [implementation](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC1155/presets/ERC1155PresetMinterPauser.sol) for more details.